lib/ansible/parsing/vault/__init__.py

```diff
@@ -75,7 +74,7 @@
 from ansible.errors import AnsibleError
 from ansible import constants as C
 from ansible.module_utils.six import PY3, binary_type
-from ansible.module_utils.six.moves import zip
+from ansible.module_utils.six.moves import zip as _zip
```

Comment:
We should disable pylint's check for this one line rather than doing this.  doing this defeats part of the purpose of having a compatibility library.